optimum/gptq/quantizer.py

```diff
@@ -601,9 +718,12 @@ def post_init_model(self, model):
         class StoreAttr(object):
             pass
 
+        if is_gptqmodel_available():
+            model, _ = hf_convert_gptq_v1_to_v2_format(model, self.bits, self.quant_linear, self.checkpoint_format, self.meta)
```

Comment:
@SunMarc  GPTQModel uses v2 as internal format for most kernels, except for IPEX but this method will auto skip conversion for IPEX. It returns converted model plus true/false if conversion happened.